src/public/virtual-keyboard.ts

```diff
@@ -98,6 +98,9 @@ export interface VirtualKeyboardKeycap {
 
   /** Name of the layer to shift to when the key is pressed */
   layer: string;
+
+  /** Open variants panel without long press and does not close automatically */
+  popover: boolean;
```

Comment:
I would probably go with something a bit more descriptive. Maybe `stickyVariantPanel`?

Reply:
That's a good name.